src/python/foglamp/translators/omf_translator.py

```diff
@@ -962,22 +973,18 @@ def omf_object_creation():
     setup_logger()
 
     prg_text = ", for Linux (x86_64)"
-    version = "1.0.24"
+    version = "1.0.28"
 
     start_message = "\n" + _module_name + " - Ver " + version + "" + prg_text + "\n" + __copyright__ + "\n"
     debug_msg_write("", "{0}".format(start_message))
     debug_msg_write("INFO", _message_list["i000002"])
-    event_loop = asyncio.get_event_loop()
-    event_loop.run_until_complete(configuration_manager.create_category(_CONFIG_CATEGORY_NAME, _DEFAULT_OMF_CONFIG,
-                                                                        _CONFIG_CATEGORY_DESCRIPTION))
-    config = event_loop.run_until_complete(configuration_manager.get_category_all_items(_CONFIG_CATEGORY_NAME))
-    _server_name = config['relay_server_name']['value']
+
+    _event_loop = asyncio.get_event_loop()
 
     initialize_plugin()
     omf_types_creation()
 
-
-    event_loop.run_until_complete(send_info_to_omf())
-    event_loop.run_until_complete(_update_statistics())
+    _event_loop.run_until_complete(send_info_to_omf())
+    _event_loop.run_until_complete(_update_statistics())
```

Comment:
yes, I'm going to close this PR and open a new one - the new one will have performance improvements and implements your current feedback (thanks for helping)